ALSpringBackend/src/main/java/com/ardaslegends/data/service/exceptions/ServiceExceptions.java

```diff
@@ -0,0 +1,4 @@
+package com.ardaslegends.data.service.exceptions;
+
+public class ServiceExceptions extends RuntimeException {
```

Comment:
We already have a ServiceException class in another branch, and since this class doesn't have any implementation here I'd just remove it so we don't have any confusion with the other branch's class

Reply:
👍 added here first, prob dont remember anymore

Reply:
Yeah let's just remove it

- [ ] Removed imposter ServiceExceptions class